client/sites/components/sites-dataviews/dataview-style.scss

```diff
@@ -1,230 +0,0 @@
-@import "@wordpress/base-styles/breakpoints";
-@import "@wordpress/base-styles/mixins";
-@import "@wordpress/dataviews/build-style/style.css";
```

Comment:
I believe we can/should retain these, as they are style from the Core side!

Reply:
This is probably why some things look very weird:

<img width="1642" alt="Screenshot 2024-11-26 at 10 08 41" src="https://github.com/user-attachments/assets/bd553a28-eaad-4e44-a49e-762ce9ffb90d">
<img width="432" alt="Screenshot 2024-11-26 at 10 09 26" src="https://github.com/user-attachments/assets/3ef4a233-8ced-43ed-98e1-6cbf32029d3c">
<img width="779" alt="Screenshot 2024-11-26 at 10 09 37" src="https://github.com/user-attachments/assets/a2f9e007-cb18-4eae-b088-52f12b80fa94">